8Knot/pages/repo_overview/visualizations/code_languages.py

```diff
@@ -42,52 +55,57 @@
                 ),
                 dcc.Loading(
                     dcc.Graph(id=f"{PAGE}-{VIZ_ID}"),
+                    style={"marginBottom": "1rem"},
+                ),
+                html.Hr(  # Divider between graph and controls
+                    style={
+                        "borderColor": "#909090",
+                        "margin": "1.5rem -1.5rem",
+                        "width": "calc(100% + 3rem)",
+                    }
                 ),
                 dbc.Form(
                     [
                         dbc.Row(
                             [
-                                dbc.Label(
```

Comment:
It looks like this section has a lot of delta that isnt really changes? If you could discard the changes and just apply the changes to the single lines that it applies to would be very helpful! For context, if we hit an issue after you leave, its helpful to know what was actually a change in this pr vs a past one

Reply:
Per slack, it seems like this is rearranging the UI to match the figma